tests/clients/test_workspace.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2018 ASI Data Science
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+from datetime import datetime
+from dateutil.tz import UTC  # type: ignore
```

Comment:
Since we're not using type checking at all, I suggest not putting `type: ignore` here.

```suggestion
from dateutil.tz import UTC
```